cypress/integration/recieve-notice/recieve-notice-checkout.spec.js

```diff
@@ -0,0 +1,154 @@
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((res) => { testData.materialType = res.id; });
+        cy.getLocations({ limit: 1 }).then((res) => { testData.location = res.id; });
+        cy.getHoldingTypes({ limit: 1 }).then((res) => { testData.holdingType = res[0].id; });
+        cy.getHoldingSources({ limit: 1 }).then((res) => { testData.holdingSource = res[0].id; });
+        cy.getInstanceTypes({ limit: 1 }).then((res) => { testData.instanceType = res[0].id; });
+        cy.getLoanTypes({ limit: 1 }).then((res) => { testData.loanType = res[0].id; });
+      })
+
+      .then(() => {
+        cy.createInstance({
+          instance: {
+            instanceTypeId: testData.instanceType,
+            title: `Pre-checkout_instance_${Number(new Date())}`,
+          },
+          holdings: [{
+            holdingsTypeId: testData.holdingType,
+            permanentLocationId: testData.location,
+            sourceId: testData.holdingSource,
+          }],
+          items: [
+            [{
+              barcode: ITEM_BARCODE,
+              missingPieces: '3',
+              numberOfMissingPieces: '3',
+              status: { name: 'Available' },
+              permanentLoanType: { id: testData.loanType },
+              materialType: { id: testData.materialType },
+            }],
+          ],
+        });
+      });
+
+    cy.loginAsAdmin({ path: settingsMenu.circulationPatronNoticePoliciesPath, waiter: NewNoticePolicyTemplate.waitLoading });
+  });
+
+  afterEach('Deleting created entities', () => {
+    CheckInActions.createItemCheckinApi({
+      itemBarcode: ITEM_BARCODE,
+      servicePointId: testData.userServicePoint,
+      checkInDate: moment.utc().format(),
+    }).then(() => {
+      cy.deleteUser(userData.id);
+      PatronGroups.deleteViaApi(patronGroup.id);
+    });
+
+    cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${ITEM_BARCODE}"` })
+      .then((instance) => {
+        cy.deleteItem(instance.items[0].id);
+        cy.deleteHoldingRecord(instance.holdings[0].id);
+        cy.deleteInstanceApi(instance.id);
+      });
+
+    CirculationRules.deleteRuleApi(testData.defaultRules);
+    NoticePolicyApi.deleteApi(testData.noticeId);
+    NoticePolicyTemplateApi.getViaApi({ query: `name=${noticePolicyTemplate.name}` }).then((templateId) => {
+      NoticePolicyTemplateApi.deleteViaApi(templateId);
+    });
+  });
+
+  it('C347621 Check that user can receive notice with multiple items after finishing the session "Check out" by clicking the End Session button', { tags: [testTypes.smoke, devTeams.vega] }, () => {
+    NewNoticePolicyTemplate.startAdding();
+    NewNoticePolicyTemplate.create(noticePolicyTemplate);
+    NewNoticePolicyTemplate.addToken(noticePolicyTemplate);
+    noticePolicyTemplate.body += '{{item.title}}';
+    NewNoticePolicyTemplate.save();
+    NewNoticePolicyTemplate.check(noticePolicyTemplate);
```

Comment:
review implementation of this method. I expect to see :
"In the right pane appears Viewing its contents.
Actions button:
Duplicate
Edit
Delete
Name notice template in the header
Patron notice template name
name
Active
Yes
Description
-
Category
name category
Email (accordion) deployed
Sublet
text
Preview button actives
Body
text, token and etc."

Reply:
it is in checkForm, which is executing in creating step